sdk/veracruz-app-lib/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
Can we rename `veracruz-app-lib` to `libveracruz`?  This was the original name of the programming support library in Veracruz that was eliminated just before you joined the project, I think.  Moreover, as this is the Rust support library, can this be put under a Rust-specific directory in the `sdk` directory, as C/C++ will also need some analogue of this library at some point?

Reply:
Done.